ts/src/coinbase.ts

```diff
@@ -1705,6 +1708,18 @@ export default class coinbase extends Exchange {
                     result[code] = account;
                 }
             }
+            {
+                const currency = this.safeValue (balance, 'currency', {});
+                const currencyId = this.safeValue (currency, 'code');
+                const code = this.safeCurrencyCode (currencyId);
+                const exponent = this.safeInteger (currency, 'exponent');
+                const currencyInfo = this.safeValue (this.currencies, code);
+                if (currencyInfo && exponent) {
+                    // Exponent corresponds to DECIMAL precision mode, the exchande precision mode is TICK_SIZE so we convert it
+                    currencyInfo.precision = Math.pow (10, -exponent).toString ();
+                    this.currencies[code] = currencyInfo;
+                }
+            }
```

Comment:
1. Why did you wrap this block by curly braces?
2. Please fix the typo `exchande` -> `exchange`
3. Don't understand your comment, could you reformulate it? An example would be great

Reply:
1 -> the `code` variable name is already taken in a `if`, so i wouldn't compile otherwise and `code` is the appropriate name following ccxt coding conventions
3 -> 
```
                    // Exponent property is expressed as a decimal exponent (ex: 8)
                    // Coinbase precisions convention is TICK_SIZE (ex: 1e-8)
                    // We convert the exponent to a string representation of the precision (ex 8 -> 1e-8)
```